README.md

```diff
@@ -1,16 +1,1831 @@
+│                                                                                                         │
+│                                                           ┌─────────┐                ┌─────────┐        │
+│                                                           │         │                │         │        │
+│                                                           │   Dan   ╞═══Read Email══►│  Erin   │        │
+│                                                           │         │           ▲    │         │        │
+│                                                           └─────────┘           ┆    └─────────┘        │
+│                                                                               With                      │
+│                                                                               Result                    │
+│                                                                  ┌─────────┐   Of         ┌─────────┐   │
+│                                                                  │         │    ┆         │         │   │
+│                                                                  │   Dan   ╞════Set DNS══►│  Erin   │   │
+│                                                                  │         │              │         │   │
+│                                                                  └─────────┘              └─────────┘   │
+│                                                                                                         │
+└─────────────────────────────────────────────────────────────────────────────────────────────────────────┘
+```
+
+## 1.3 A Note On Serialization
+
+The JSON examples below are given in [DAG-JSON], but UCAN Task is actually defined as IPLD. This makes UCAN Task agnostic to encoding. DAG-JSON follows particular conventions around wrapping CIDs and binary data in tags like so:
+
+### CID
+
+```json
+{"/": "Qmf412jQZiuVUtdgnB36FXFX7xg5V6KEbSJ4dpQuhkLyfD"}
+
+### Bytes
+
+```json
+{"/": {"bytes": "s0m3Byte5"}}
+```
+
+This format help disambiguate type information in generic [DAG-JSON] tooling. However, your presentation need not be in this specific format, as long as it can be converted to and from this cleanly. As it is used for the signature format, [DAG-CBOR] is RECOMMENDED.
+
+## 1.4 Signatures
+
+All payloads described in this spec MUST be signed with a [Varsig].
+
+# 2 High-Level Concepts
+
+## 2.1 Roles
+
+Task adds two new roles to UCAN: invoker and executor. The existing UCAN delegator and delegate principals MUST persist to the invocation.
+
+| UCAN Field | Delegation                             | Task                            |
+| ---------- | -------------------------------------- | ------------------------------- |
+| `iss`      | Delegator: transfer authority (active) | Invoker: request task (active)  |
+| `aud`      | Delegate: gain authority (passive)     | Executor: perform task (active) |
```

Comment:
IIRC you didn't like these terms back in December. If that's still the case, I'm happy to change them!

Reply:
I can't recall the details, but I don't mind the these terms